crates/api/src/community/pending_follows/count.rs

```diff
@@ -0,0 +1,25 @@
+use actix_web::web::{Data, Json, Query};
+use lemmy_api_common::{
+  community::{GetCommunityPendingFollowsCount, GetCommunityPendingFollowsCountResponse},
+  context::LemmyContext,
+  utils::is_mod_or_admin,
+};
+use lemmy_db_views::structs::LocalUserView;
+use lemmy_db_views_actor::structs::CommunityFollowerView;
+use lemmy_utils::error::LemmyResult;
+
+pub async fn get_pending_follows_count(
```

Comment:
This is good I think. We might want to consider unifying the terminology of either this or `get_unread_registration_application_count` to match. (IE one is called *unread*, the other is called *pending*), but its not a big deal.

I thought about combining them, but IMO its not worth it for the same reason that the API responses should be separate: one is done by admins only, while the other is done by community mods.

Reply:
Pending makes more sense imo, because it needs to be `approved`, not `read`.